automated-comments/go/two_fer/missing_package_comment.md

```diff
@@ -0,0 +1 @@
+Go's community style says that package `twofer` should have a comment. For more details, have a look at the comments section below.
```

Comment:
Rather than "Go's community style" I would prefer _It's idiomatic in Go_. I don't really know what a 'community style' is.

Rather than "have a look" I would prefer "see", as this is shorter.

Reply:
> It's idiomatic in Go for package to have a comment. For more details see the comments section below.